galgebra/mv.py

```diff
@@ -33,6 +33,82 @@
 
 half = Rational(1, 2)
 
+
+# Add custom settings to the builtin latex printer
+_LatexPrinter._default_settings.update({
+    'galgebra_mv_fmt': 1
+})
+_StrPrinter._default_settings.update({
+    'galgebra_mv_fmt': 1
+})
```

Comment:
Is this a good place to init the setting?

Reply:
I spoke to @asmeurer about this on gitter here: https://gitter.im/sympy/sympy?at=5ebed3cf13878c30b57fe506

For now, this looks like the best approach.

Reply:
Worth noting that with this change, users can now use `sympy.init_printing(galgebra_mv_fmt=...)` to change the setting globally.